pkgs/development/tools/misc/src-cli/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "src-cli";
+  version = "4.0.0";
+
+  src = fetchFromGitHub {
+    owner = "sourcegraph";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-cd96ddFtPOMATI0MEMFH399/y7SZeASetVhvRt0P38A=";
+  };
+
+  vendorSha256 = "sha256-ll/Q7hZJngOGG/Jl79KvEIHQ+ARDxFD7KHY3nSokRi0=";
+
+  subPackages = [ "cmd/src" ];
+
+  CGO_ENABLED = 0;
+
+  ldflags = [ "-X github.com/sourcegraph/src-cli/internal/version.BuildTag=${version}" ];
+
+  meta = with lib; {
```

Comment:
Could you add `changelog = "https://github.com/sourcegraph/src-cli/blob/${version}/CHANGELOG.md";`?